src/test/suites/product/ProductFiltersTest.js

```diff
@@ -0,0 +1,44 @@
+import { Selector } from 'testcafe'
+
+fixture("product")
+    .page(process.env.TEST_BASE_URL);
+
+test("Apply Apple and Samsung Filters", async (t) => {
+
+    // As of now this test is set to filter by apple or samsung
+    // to filter for any other device manufaurers modify the regex for the `checkBoxes` selector
+    await t.wait(2000);
```

Comment:
please identify a better way to do away with the hardcoded wait.
can there be an element which could be waited upon?